README.md

```diff
@@ -31,6 +31,7 @@ If you want to see the code - go to this [section](#tldr)!
 	2. [Closed Captioning / SAP Settings button](#closed-captioning--sap-settings-button)
 	3. [Using the 4 Custom Sidebar buttons](#using-the-4-custom-sidebar-buttons)
 	4. [Setting the LIVE indicator’s tint color](#setting-the-live-indicators-tint-color)
+	5. [Animations customization and disabling](#animations-customization-and-disabling)
```

Comment:
change:
[Animations customization and disabling](#animations-customization-and-disabling)

to:
[Animations Customization and Disabling](#animations-customization-and-disabling)